Source/ACE.Server/Managers/PlayerManager.cs

```diff
@@ -122,6 +131,16 @@ public static OfflinePlayer GetOfflinePlayer(uint guid)
             return null;
         }
 
+        public static int GetOfflineCount()
```

Comment:
This is not thread safe. The thread safe way to do this is: PlayerManager.GetAllOffline().Count

This is how it's done in "serverstatus"

Reply:
modified this function to use the correct method